src/Controller/Api/PostController.php

```diff
@@ -24,6 +23,7 @@
 use Sonata\NewsBundle\Mailer\MailerInterface;
 use Sonata\NewsBundle\Model\Comment;
 use Sonata\NewsBundle\Model\CommentManagerInterface;
+use Sonata\NewsBundle\Model\Post;
```

Comment:
It seems like a BC break. Is this a bugfix? In that case, it should be done in other PR.

Reply:
Technicly it is BC-break and it is bugfix, anyway it can be change like 'REST' routing type. This will change only docblock, Do we really need this in seperate PR when all RestApi system is rebuilld?

Reply:
IMO we should consider this as a bugfix

Reply:
If this change fixes something what will start working without further modifications, we should create a separate PR.

Reply:
@phansys This is only bug on documentation. It will be working anyway, with or without this change.

@jordisala1991 Do we really need seperate PR for it?

Reply:
>@phansys This is only bug on documentation. It will be working anyway, with or without this change.

Please, note that the documentation is not just intended to be read by humans. It generates definitions compatible with the [Open API Specification](https://swagger.io/docs/specification/2-0/basic-structure/), which is also intended to build API clients (like the sandbox exposed in the HTML documentation).
Wrong specification produces wrong clients.

Reply:
@phansys Can you do it? You have better knowlage why and how this should be fix. Also will be nice to make [`Update Swagger definition in other to be compatible with string identifiers`](https://github.com/sonata-project/SonataUserBundle/pull/1198) change in other bundles to make RestApi upgrade more complex. 

WDYT?

Reply:
I'll try, but be patient. I'm very busy these days. I'll try to get some ReST :trollface: this weekend, and in the meantime I'll see if I can provide some PRs.

Reply:
This PR can be provided independent of my work, so dont worry be happy .